src/components/TableCell/index.js

```diff
@@ -0,0 +1,32 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { fonts } from '../../styles'
+
+export const TableCell = styled.div`
+    ${props => props.flex && `
+        flex: ${props.flex}
+    `}
+    ${props => props.width && `
+        width: ${props.width}
+    `}
+    padding-top: 12px;
+    padding-left: 10px;
+    padding-right: 10px;
+    padding-bottom: 8px;
```

Comment:
`padding: 12px 10px 8px 10px`, no?

Reply:
I actually prefer writing it more verbosely, then you don't have to remember the order